Environment/covid-19_stats.15m.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/zsh
```

Comment:
Please use /bin/bash or another supported shell.  zsh scripts are not supported as we can't lint them.

Reply:
updated the shebang and needed to update one addition line because of a change to the behavior of grep. Thanks for the tip, I'll make sure any future scripts are bash from the start. As zsh is now the default shell, would zsh linting be possible in the future? I'm not familiar enough to know why it wouldn't be lint-able